pkg/cmd/admin/node/schedulable.go

```diff
@@ -20,7 +20,7 @@ func (s *SchedulableOptions) Run() error {
 	errList := []error{}
 	var printer kubectl.ResourcePrinter
 	for _, node := range nodes {
-		if node.Spec.Unschedulable == !s.Schedulable {
+		if node.Spec.Unschedulable == s.Schedulable {
```

Comment:
Just to be sure I got it.  This is what we're shooting for right?

 | node.Spec.Unschedulable == true | node.Spec.Unschedulable == false |
------------ | ------------- | ----------
 s.Schedulable == true | change to false | do nothing | 
 s.Schedulable == false | do nothing | change to true |